src/frontend/screens/Settings/ProjectConfig.tsx

```diff
@@ -86,6 +86,12 @@ const ProjectConfig = () => {
     ]);
   }, [didError, t]);
 
+  React.useEffect(() => {
+    config.practiceModeBarToggle!(false);
```

Comment:
is the `!` supposed to be there?

Reply:
ah nvm i see it's using [this operator](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#non-null-assertion-operator-postfix-)